lib/trino-orc/src/main/java/io/trino/orc/stream/LongInputStreamV1.java

```diff
@@ -64,7 +64,7 @@ private void readValues()
             }
 
             // convert from 0 to 255 to -128 to 127 by converting to a signed byte
-            delta = (byte) delta;
+            delta = Integer.valueOf(this.delta).byteValue();
```

Comment:
Is there other way to suppress this without changing actual code?

Reply:
I can add an explicit `@SuppressWarnings`, obviously, but it might make the code even less clear (if we insist adding it on the statement level, not the method level). In general I'm inclined to think that this is a real false positive on Error Prone's side, perhaps we should report it (it won't be of immediate help, though, as the Error Prone releases happen about twice a year or so).

Reply:
I tried various ways to handle this, and this was the least worst way I could come up with

Reply:
Regardless what we end up doing here, this is a bug in the check. Please report it. See https://github.com/google/error-prone/issues/2102 for example report.

Reply:
FWIW, https://github.com/google/error-prone/issues/2196

Reply:
The response from the Error Prone team is that this result is deliberate and the check for casts was explicitly added as a result of their analysis of the cases they found in their code bases. I think I have an alternative which is more directly solving the problem than a cast.